client/web/src/enterprise/codeintel/configuration/components/inference-form/CommandInput.tsx

```diff
@@ -67,6 +67,7 @@ export const CommandInput: React.FunctionComponent<CommandInputProps> = React.me
     const extensions = useMemo(
         () => [
             staticExtensions,
+            [EditorView.lineWrapping].filter(() => readOnly),
```

Comment:
```suggestion
            readonly ? EditorView.lineWrapping : [],
```

is a bit more ideomatic in the CodeMirror world I think.

Reply:
ah yes, forgot that JS/TS has a ternary operator. thank you!